home.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Coder's Community</title>
+    <link rel="stylesheet" href="style/style.css" />
+  </head>
+  <body>
+    <!-- Container Starts -->
+    <div class="container">
+      <!-- Navigation Starts -->
+      <nav>
+
+        <div class="logo">
+          <img class="logo-img" src="/media/png/unnamed.png" alt="" />
```

Comment:
- IN src you need to add `./` or just don't add anything then only this will work when hosted on github pages

Reply:
```
Should I remove `../` and `/` from CSS file too?
/* Variables and Font-Faces */
@font-face {
  font-family: "Druk Wide Bold";
  src: url(../media/font/Druk\ Wide\ Bold.ttf);
}
@font-face {
  font-family: "Work Sans";
  src: url("../media/font/WorkSans-Regular.ttf"),
    url("../media/font/WorkSans-SemiBold.ttf");
}

```

Reply:
`../` will not make a issue but just `/` without a dot is not supported in many cases

Reply:
okayy resolve this issue right now...